graphdatascience/graph/graph_object.py

```diff
@@ -230,3 +231,194 @@ def __repr__(self) -> str:
             "memoryUsage",
         ]
         return f"{self.__class__.__name__}({self._graph_info(yields=yield_fields).to_dict()})"
+
+    def visualize(
```

Comment:
i like the idea of this.
maybe we can have different backends to be used?
pyvis/nvl?  (with default to nvl)

should have a thought on the dependency. Likely similar optional dependency as for networkx